README.md

```diff
@@ -240,12 +240,12 @@ All properties are mandatory.
 }
 ```
 
-Only the `topic` and properties are mandatory
+Only the `topic` and `payload` properties are mandatory.
```

Comment:
Yes you're right. If omitted it will default to `empty`

https://github.com/mqttjs/mqtt-packet/blob/master/writeToStream.js#L221